data/preprocessing/gadm/Makefile

```diff
@@ -1,34 +1,30 @@
-.PHONY: geo_region_table combine-gadm-file
+.PHONY: upload_results
+SHELL := /bin/bash
 
-# results of data processing
-DATADIR=data
-# intermediate files
-WORKDIR=data/tmp
+WORKDIR=data
 
-all: checksum
+all: upload_results
 	make clean-workdir
 
+upload_results: checksum
+	aws s3 cp $(WORKDIR)/geo_region.zip $(AWS_S3_BUCKET_URL)/processed/geo_region/
+	#rm $(WORKDIR)/geo_region.zip $(WORKDIR)/geo_region.csv
```

Comment:
should we keep this comment here?

Reply:
deleted it because we clean all the data files after finishing automatically anyway